cfme/cloud/provider.py

```diff
@@ -114,13 +116,14 @@ class Provider(Pretty, CloudInfraProvider):
         {version.LOWEST: form_buttons.FormButton("Save Changes"),
          '5.5': form_buttons.FormButton("Save changes")})
 
-    def __init__(self, name=None, credentials=None, zone=None, key=None):
+    def __init__(self, name=None, credentials=None, zone=None, key=None, project=None):
         if not credentials:
             credentials = {}
         self.name = name
         self.credentials = credentials
         self.zone = zone
         self.key = key
+        self.project = project
```

Comment:
I _think_ this should be on the ```GCEProvider``` class as it only corresponds to GCE, we _may_ have some others that we made a mistake on in the past...```zone```??? However, I think this is the right way to do it.